docs/user/integration/sample-app/README.md

```diff
@@ -0,0 +1,120 @@
+# Integrate Custom Instrumentation App
+
+## Overview
+
+| Category| |
+| - | - |
+| Signal types | traces, metrics, logs |
+| Backend type | custom in-cluster, third-party remote |
+| OTLP-native | yes |
+
+Learn how to instrument your custom Golang application using the [OTel SDK](https://opentelemetry.io/docs/languages/) and exporting metrics and trace data using the OTel SDK. The sample application will be configured to push trace and metric data using OTLP to the collector that's provided by Kyma, so that they are collected together with the related Istio trace data.
+
+For examples using the OTel-SDK in a different langue, please refer to the official [OTel guides](https://opentelemetry.io/docs/languages/) and the [OTel demo app](./../opentelemetry-demo/).
+
+![setup](./../assets/sample-app.drawio.svg)
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- The [Telemetry module](../../README.md) is [added](https://kyma-project.io/#/02-get-started/01-quick-install)
+- [Kubectl version that is within one minor version (older or newer) of `kube-apiserver`](https://kubernetes.io/releases/version-skew-policy/#kubectl)
+
+## Exploring the Sample App
+
+The sample app is a small webserver written in Golang, which exposes to endpoints `forward` and `terminate`. When calling the endpoints via HTTP, metrics are getting counted up and spans are emmitted using OTLP exporters. Furthermore, structured logs are written to `stdout`.
+<!-- markdown-link-check-disable-next-line -->
+The application is located in the [`telemetry-manager`](https://github.com/kyma-project/telemetry-manager) repo in the folder [`docs/user/integration/sample-app`](https://github.com/kyma-project/telemetry-manager/tree/main/docs/user/integration/sample-app).

```

Comment:
cant we just give the link directly ? Why putting info about repo and folder

Reply:
```suggestion
The application is located in the [`telemetry-manager`](https://github.com/kyma-project/telemetry-manager/tree/main/docs/user/integration/sample-app) repository.

```